.slurmci/cpu-init.sh

```diff
@@ -7,12 +7,9 @@
 set -euo pipefail
 set -x #echo on
 
-export PATH="${PATH}:${HOME}/julia-1.2/bin"
 export JULIA_DEPOT_PATH="$(pwd)/.slurmdepot/cpu"
 export OPENBLAS_NUM_THREADS=1
 
-module load openmpi/3.1.4
+module load julia/1.2.0 openmpi/4.0.1
```

Comment:
Just a note that we ran into issues with openmpi/4.0.1, which is why we downgraded to 3.1.4. (see https://github.com/climate-machine/CLIMA/pull/381#issuecomment-523306892)

Reply:
I saw that, but didn't see any such errors when testing. At some point, using a CUDA-aware MPI is necessary...

Reply:
Just wanted to make sure you were aware of it. Something we need to keep an eye on.